builders/promote.yaml

```diff
@@ -244,17 +244,17 @@ steps:
   - '-'
 
 #
-# jwks
+# jwks-updater
 #
-- id: 'promote-jwks'
+- id: 'promote-jwks-updater'
   name: 'gcr.io/google.com/cloudsdktool/cloud-sdk:307.0.0-alpine'
   args:
   - 'bash'
   - '-eEuo'
   - 'pipefail'
   - '-c'
   - |-
-    gcloud run services update-traffic "jwks" \
+    gcloud run services update-traffic "jwks-updater" \
```

Comment:
don't we need to update the terraform as well?

Reply:
Yea, I decided to just rename the binary to match the deploy instead of editing all these files.